### ISSUE TITLE
[Bug] Bluetooth not working on macOS 15.0 (Sequoia)

### ISSUE THREAD

**Issue by shiruken** [shiruken/hackintosh#24]
Since updating (98c10a5111b807c8b5f43691e4e9a2e3ff0b09f9) to macOS 15.0 (Sequoia), Bluetooth has not been working. Bluetooth cannot be enabled in Control Center and is unresponsive when the toggle is enabled in System Settings.

**Comment by shiruken** [comment]
It appears that the most recent version (v2.6.8) of `BlueToolFixup.kext` from [BrcmPatchRAM](https://github.com/acidanthera/BrcmPatchRAM) only targets macOS 14 (Sonoma). A [pull request](https://github.com/acidanthera/BrcmPatchRAM/pull/33) has recently been merged to add support for macOS 15 (Sequoia). A pre-release build of the kext can be downloaded from the artifacts for the [CI workflow](https://github.com/acidanthera/BrcmPatchRAM/actions).

**Comment by bobmarinho** [comment]
I updated to Sequoia and got my bluetooth and wifi not working anymore.
I installed that pre-release version and my bluetooth got back to work, but the wifi still not working. Are you using itlwm with Heliport?
Before the update to Sequoia I was using Airportitlwm and it was working natively, seems for now I need to switch to itlwm + Heliport.

Thanks

**Comment by shiruken** [comment]
> Are you using itlwm with Heliport?

Yup. I've only ever used itlwm so I can't comment on AirportItlwm compatibility.
